README.md

```diff
@@ -138,11 +138,11 @@ Activating this enables the following behaviors:
 
 When not in window management mode, pressing `Super` + `<Direction>` will shift window focus to a window in the given direction. This is calculated based on the distance between the center of the side of the focused window that the window is being shifted from, and the opposite side of windows surrounding it.
 
-Switching focus to the left will calculate from the center of the east side of the focused window, to the center of the west side of all other windows. The window with the least distance is the window we pick.
+Switching focus to the left will calculate from the center of the east side of the focused window to the center of the west side of all other windows. The window with the least distance is the window we pick.
 
 ### Launcher
 
-The launcher is summoned with `Super` + `/`. The search list displays matching windows based on their window name and title, and applications on the system which can be launched. The arrow keys are used to select an application or window from the search list. If it is a window, the selected window will be visually highlighted with an overlay, in the event that two windows have the same name and title. Pressing `Return` on a window will bring that window to focus, switching to its workspace, and unminimizing it if it was minimized.
+The launcher is summoned with `Super` + `/`. The search list displays matching windows based on their window name and title, and applications on the system which can be launched. The arrow keys are used to select an application or window from the search list. If it is a window, the selected window will be visually highlighted with an overlay, if two windows have the same name and title. Pressing `Return` on a window will bring that window to focus, switching to its workspace, and unminimizing it if it was minimized.
```

Comment:
The usage of `in the event that` wasn't gramatically incorrect before, but the change to `if` is fine grammatically. However, that clause is dependent, so the comma before it should be removed.

Furthermore, the actual sentence was incorrect before, and still is: the Launcher will always highlight the selected window, regardless of whether it's unique. `if` could be changed to `in case` to clarify that we're just referring to the anticipated use case of the feature, but I'd lean toward removing that clause (end the sentence after `overlay`.)